util/util_windows.go

```diff
@@ -14,18 +14,23 @@ import (
 
 var utilLogger = logging.GetLogger("util")
 
-// RunCommand XXX
+// RunCommand runs command (in two string) and returns stdout, stderr strings and its exit code.
 func RunCommand(command, user string) (string, string, int, error) {
 	var outBuffer, errBuffer bytes.Buffer
-
 	wd, err := os.Getwd()
 	if err != nil {
 		return "", "", -1, err
 	}
-	cmd := exec.Command("cmd", "/c", "pushd "+wd+" & "+command)
+	cmdArgs := []string{"cmd", "/c", "pushd " + wd + " & " + command}
```

Comment:
please use `cmd.Dir = wd` if you don't have expricitly reason to use "pushd" command.
```diff
diff --git a/util/util_windows.go b/util/util_windows.go
index 8713ea9..ed1259d 100644
--- a/util/util_windows.go
+++ b/util/util_windows.go
@@ -22,10 +22,11 @@ func RunCommand(command, user string) (string, string, int, error) {
 	if err != nil {
 		return "", "", -1, err
 	}
-	cmd := exec.Command("cmd", "/c", "pushd "+wd+" & "+command)
 	if user != "" {
 		utilLogger.Warningf("RunCommand ignore option: user = %q", user)
 	}
+	cmd := exec.Command("cmd", "/c", command)
+	cmd.Dir = wd
 	cmd.Stdout = &outBuffer
 	cmd.Stderr = &errBuffer
```